pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -44,15 +46,16 @@ func NewReconcileAutoImport(
 	client client.Client,
 	kubeClient kubernetes.Interface,
 	informerHolder *source.InformerHolder,
-	recorder events.Recorder,
+	globalRecorder events.Recorder,
+	mcRecorder kevents.EventRecorder,
```

Comment:
it is difficult to understand why we need 2 recorder here if there is no comments.
do we consider to merge the two recorders ?  
the events are in different namespaces, how about use one method here and diff them in the method ?